src/util/misc.ts

```diff
@@ -316,7 +316,7 @@ export function sanitizeString(str: string) {
 // We will need the encoded URL on article load so that we can set the hrefs of anchor tag correctly,
 // but we must not encode the '/' character or else relative links may fail
 export function encodeArticleIdForZimHtmlUrl(articleId: string) {
-  return articleId && encodeURIComponent(articleId).replace(/%2F/g, '/');
+  return articleId && encodeURIComponent(articleId[0] === '/' ? `./${articleId}` : articleId).replace(/%2F/g, '/');
```

Comment:
If `articleId` is empty string it will fail probably. This test should better be done with `startsWith()` or using a regex.

Reply:
right, will change that